lib/net.js

```diff
@@ -1379,6 +1397,14 @@ Server.prototype.listen = function() {
   var options = normalized[0];
   const cb = normalized[1];
 
+  if (this[serverListenHasBeenCalled]) {
+    var er = new Error('listen method has been called more than once.');
+    er.code = 'EALREADY';
```

Comment:
This is in line with what libuv returns for client sockets but using EALREADY instead of EINVAL (the standard POSIX error for trying to rebind) is one of my little regrets there.

Thoughts, anyone?  Consistency or conformity?